crypto/keypair.go

```diff
@@ -13,6 +13,12 @@ type KeyType = string
 const Ed25519Type KeyType = "ed25519"
 const Sr25519Type KeyType = "sr25519"
 
+const PublicKeyLength int = 32
+const SeedLength int = 32
+const Ed25519PrivateKeyLength int = 64
+const Sr25519PrivateKeyLength int = 32
+const SignatureLength int = 64
+
```

Comment:
These don't seem to have anything to do with the abstract keypair (ie. these don't seem to be enforced as part of the interface). It might be preferential to move these in the implementations, providing specific values for each type (eg. `Ed25519PublickeyLength`).

Reply:
You could also move them to subpackages to reduce naming conflicts

Reply:
fair, I'll move them to the respective files